tests/integration_tests/test_pytorch_distributed.py

```diff
@@ -351,3 +351,23 @@ def test_updates_properties(storage_mode: str) -> None:
             [getattr(trial, p) for p in property_names]
 
         dist.barrier()  # type: ignore[no-untyped-call]
+
+@pytest.mark.filterwarnings("ignore::optuna.exceptions.ExperimentalWarning")
```

Comment:
A nit, but could you add a blank line for `black`?

```suggestion


@pytest.mark.filterwarnings("ignore::optuna.exceptions.ExperimentalWarning")
```

Reply:
I think I fixed it. Let me see what happens 👀

Reply:
Thanks!